pkg/restore/clusterrolebinding_action_test.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2018, 2019 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package restore
+
+import (
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
+	"github.com/vmware-tanzu/velero/pkg/test"
+)
+
+func TestClusterRoleBindingActionAppliesTo(t *testing.T) {
+	action := NewClusterRoleBindingAction(test.NewLogger())
+	actual, err := action.AppliesTo()
+	require.NoError(t, err)
+	assert.Equal(t, velero.ResourceSelector{IncludedResources: []string{"clusterrolebindings"}}, actual)
+}
+
+// 3 cases: no mapping - return rolebinding unchanged, mapping enabled change namespace
+func TestClusterRoleBindingActionExecute(t *testing.T) {
+	tests := []struct {
+		name             string
+		namespaces       []string
+		namespacemapping map[string]string
+		expected         []string
+	}{
+		{
+			name:             "namespace mapping disabled",
+			namespaces:       []string{"foo"},
+			namespacemapping: map[string]string{},
+			expected:         []string{"foo"},
+		},
+		{
+			name:             "namespace mapping enabled",
+			namespaces:       []string{"foo"},
```

Comment:
It would be good to test that a namespace not included in the mapping is left unchanged.